uo_sys_framework/ACREModule/cfgFunctions/common/fn_addRack.sqf

```diff
@@ -0,0 +1,148 @@
+
+private _rackType = GETVAR(_object,UO_FW_ACRE_RACK_TYPE,"ACRE_VRC103");
+private _rackNames = GETVAR(_object,UO_FW_ACRE_NAMES,["Default Rackname","Rack"]); 
+private _rackNameLong = _rackNames select 0; private _rackNameShort = _rackNames select 1;
+private _mountRadio = GETVAR(_object,UO_FW_ACRE_MOUNTED_RADIO,false);
+private _mountedRadio = "ACRE_VRC103";
+private _radioRemovable = GETVAR(_object,UO_FW_ACRE_RACK_TYPE,false);
+private _whitelistedPositions = GETVAR(_object,UO_FW_ACRE_WHITELISTED_POSITIONS,["inside"]);
+private _blacklistedPositions = GETVAR(_object,UO_FW_ACRE_BLACKLISTED_POSITIONS,[]);
+private _faction = GETVAR(_object,UO_FW_ACRE_FACTION_PRESET,civilian);
+
+_condition = {
+	CBA_missionTime > 0
+};
+
+if((missionNamespace getVariable ["UO_FW_ACRE_SCRAMBLE_ENABLED", false] || missionNamespace getVariable ["UO_FW_ACRE_BABEL_ENABLED", false]) && {isNil "_faction"}) then {
+	(format ["(ACRE_SETUP_MODULE) [WRN]: RADIO SCRAMBLING OR BABEL ARE ENABLED BUT %1 HAS NO ASSIGNED FACTION", _object]) call UO_FW_FNC_DebugMessage;
+};
+
+if(isNil "_rackType") then {
+	(format ["(ACRE_SETUP_MODULE) [ERR]: RACK CLASSNAME: %1 DOES NOT EXIST OR A TYPO EXISTS VERIFY YOUR RACK CLASSNAME: %1", _rackType]) call UO_FW_FNC_DebugMessage;
+	(format ["(ACRE_SETUP_MODULE) [ERR]: VERIFY YOUR RACK CLASSNAME: %1", _rackType]) call UO_FW_FNC_DebugMessage;
+	_rackType = "ACRE_VRC103";
+	(format ["(ACRE_SETUP_MODULE) [ERR]: DEFAULTING TO CLASSNAME: %1", _rackType]) call UO_FW_FNC_DebugMessage;
+};
+
+_args pushBackUnique _rackType;
+
+if(isNil "_radioRemovable") then {
+	(format ["(ACRE_SETUP_MODULE) [ERR]: RADIO REMOVEABILITY HAS NOT BEEN SPECIFIED: %1", _radioRemovable]) call UO_FW_FNC_DebugMessage;
+	_radioRemovable = false;
+	(format ["(ACRE_SETUP_MODULE) [ERR]: DEFAULTING TO: %1", _radioRemovable]) call UO_FW_FNC_DebugMessage;
+};
+
+_args pushBackUnique _radioRemovable;
+
+if(_mountRadio) then {
+	switch (_rackType) do {
+		case "ACRE_SEM90": {_mountedRadio = "ACRE_SEM70";};
+		case "ACRE_VRC103": {_mountedRadio = "ACRE_PRC117F";};
+		case "ACRE_VRC110": {_mountedRadio = "ACRE_PRC152";};
+		case "ACRE_VRC111": {_mountedRadio = "ACRE_PRC148";};
+		case "ACRE_VRC64": {_mountedRadio = "ACRE_PRC77";};
+		default {
+			(format ["(ACRE_SETUP_MODULE) [ERR]: NFB ERROR ENCOUNTERED"]) call UO_FW_FNC_DebugMessage;
+			(format ["(ACRE_SETUP_MODULE) [ERR]: VERIFY YOUR RACK CLASSNAME: %1", _rackType]) call UO_FW_FNC_DebugMessage;
+		};
+	};
+} else {_mountedRadio = "";};
+
+_args pushBackUnique _mountedRadio;
+
+if(isNil "_rackNameLong" || isNil "_rackNameShort") then {
+	(format ["(ACRE_SETUP_MODULE) [ERR]: A RACKNAME IS NIL, LONG: %1 SHORT: %2", _rackNameLong, _rackNameShort]) call UO_FW_FNC_DebugMessage;
+	(format ["(ACRE_SETUP_MODULE) [ERR]: DEFAULTING TO LONGNAME: ""DEFAULT RACK NAME: FIX ME"" AND SHORTNAME: ""FIX"""]) call UO_FW_FNC_DebugMessage;
+	_rackNameLong = "DEFAULT RACK NAME: FIX ME";
+	_rackNameShort = "FIX";
+};
+
+_args pushBackUnique _rackNameLong;
```

Comment:
`pushBackUnique` for both of the rack names would mean it wouldn't take a Long and Short name that are equal/the same.